compiler/rustc_mir_build/src/builder/mod.rs

```diff
@@ -438,9 +436,7 @@ macro_rules! unpack {
     }};
 }
 
-///////////////////////////////////////////////////////////////////////////
 /// the main entry point for building MIR for a function
```

Comment:
Might as well make it a proper sentence.
```suggestion
/// The main entry point for building MIR for a function.
```